tsconfig.json

```diff
@@ -1,17 +1,17 @@
 {
   "compilerOptions": {
     "target": "ES2021",
-    "allowJs": true,
     "esModuleInterop": true,
     "module": "Node16",
     "moduleResolution": "Node16",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "skipLibCheck": true,
     "skipDefaultLibCheck": true,
-    "declaration": true,
```

Comment:
The `declaration` files are helpful for libraries as they provide intellisense for compiled code. However, with an application they are not super useful.